bloom-housing/bloom
# fix: missing state data

# Pull Request Template

## Issue Overview

This PR addresses #3291 


## Description

Through QA, it was found that "Leasing Agent State" was missing. This PR adds that missing state column along with other recorded state fields. 

Note: will wait for confirmation from @sarahlazarich  before merging.

## How Can This Be Tested/Reviewed?

This can by tested by ensuring at least one listing has the address filled out for leasing agent address, mailing address, and pickup address. Then, click export and ensure the column is present.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
